src/components/CircularProgressBar/index.tsx

```diff
@@ -1,84 +1,85 @@
 import React from 'react';
+type Props = {
+  sqSize: number;
+  strokeWidth: number;
+  public: number;
+  private: number;
+  class: string;
```

Comment:
change this to something else than `class`

Reply:
done, will push in seconds